vars/publishToNuget.groovy

```diff
@@ -31,13 +32,16 @@ void call(Map args = [:]) {
             overwrite: true
             )
         }
+
+    sh """
+        dotnet pack ${WORKSPACE}/${args.solutionFilePath} --configuration Release --no-build
+        find src -name OpenSearch*.nupkg > ${WORKSPACE}/nupkg.txt
```

Comment:
Just wondering do we have to use a file instead of saving in var?

Reply:
Makes it easier to read a file in groovy that var.

Reply:
ok sure.